src/components/layout.js

```diff
@@ -66,6 +66,12 @@ const Layout = ({ children, bg }) => {
             height: '100%',
             overflowX: 'hidden',
           },
+          '.Toastify__toast-body': {
+            margin: 'auto 8px',
+          },
+          '.Toastify__toast--success': {
+            background: useThemeUI().theme.colors.primary,
```

Comment:
Hooks may only be called at the top level of a component. `@theme-ui/css` would probably be easier to implement themed global styles.